config/rbac/leader_election_role.yaml

```diff
@@ -8,8 +8,6 @@ rules:
       - ""
```

Comment:
Keep the resource name for `get`, `patch`, `update`. Use this:
```
metadata:
  name: controller-leader-election-role
rules:
- apiGroups:
  - ""
  resources:
  - configmaps
  verbs:
  - list
  - watch
  - create
- apiGroups:
  - ""
  resources:
  - configmaps
  resourceNames:
  - aws-load-balancer-controller-leader
  verbs:
  - get
  - update
  - patch
```

Reply:
@fawadkhaliq
I restricted them via https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/1555.